vars/essentialsTest.groovy

```diff
@@ -2,6 +2,7 @@ def call(Map params = [:]) {
     def baseDir = params.containsKey('baseDir') ? params.baseDir : "."
     def metadataFile = params.containsKey('metadataFile') ? params.metadataFile : "essentials.yml"
     def labels = params.containsKey('labels') ? params.labels : "docker && highmem"
+    def pluginEvaluationOutcome = params.get("pluginEvaluationOutcome") ?: "failOnInvalid"
```

Comment:
The argument name cannot be understood bu a user without reading docs.
I would rather prefe...

```yaml
metadata:
   testPluginResolution:
     skipOnUnmetDependencies: true
```

with defaulting to `failOnInvalid`

Reply:
Absolutely, that makes it much more nicer, will update the PR